### PR TITLE
[fix] prevent requests being sent to python model until model is full…

### PR DESCRIPTION
**Pull request by siddvenk**
…y loaded and warmup request has completed

## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: engines/python/src/main/java/ai/djl/python/engine/PyProcess.java [hunk 1]
```diff
@@ -188,6 +190,7 @@ synchronized void startPythonProcess() {
     }
 
     synchronized void stopPythonProcess(boolean error) {
+        modelLoaded = false;
```
Comment (frankfliu) [inline engines/python/src/main/java/ai/djl/python/engine/PyProcess.java:193]: I feel move this to startPythonProcess is more readable

File: engines/python/src/main/java/ai/djl/python/engine/PyProcess.java [hunk 2]
```diff
@@ -237,6 +240,10 @@ boolean isStopped() {
         return !started;
     }
 
+    boolean isModelLoaded() {
```
Comment (frankfliu) [inline engines/python/src/main/java/ai/djl/python/engine/PyProcess.java:243]: We can combine isStopped() and isModelLoaded() into `isRead()`

File: engines/python/src/main/java/ai/djl/python/engine/PyProcess.java (hunk 1 above)
Comment (siddvenk) [reply engines/python/src/main/java/ai/djl/python/engine/PyProcess.java]: done

File: engines/python/src/main/java/ai/djl/python/engine/PyProcess.java (hunk 2 above)
Comment (siddvenk) [reply engines/python/src/main/java/ai/djl/python/engine/PyProcess.java]: done

**Review by frankfliu — APPROVED** [review]

**Pull request merged**